product-mini/platforms/zephyr/simple-http/wasm-apps/http_get.c

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
Is this file ported from other project? If yes, normally we add it into <wamr_root>/ATTRIBUTIONS.md, and add a license file in the related folder.
see:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/ATTRIBUTIONS.md

Reply:
The sample is highly inspired from : https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/net/sockets/http_get/src/http_get.c

Add I see that zephyr already appear in the [ATTRIBUTIONS.md](https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/ATTRIBUTIONS.md)

What should we do ?